wire-ios-sync-engine/Source/UserSession/ZMUserSession/ZMUserSession.swift

```diff
@@ -302,6 +302,8 @@ public final class ZMUserSession: NSObject {
         notificationDispatcher.tearDown()
         callCenter?.tearDown()
 
+        contextStorage.clear()
```

Comment:
This is called implicitly on object release, but retaining the `var` had the original purpose to call `clear()` explicitly.